### DISCUSSION TITLE
deploying-asp-net-core-applications-to-kubernetes-part-11-avoiding-downtime-in-rolling-deployments-by-blocking-sigterm/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#29] [Announcements]
# deploying-asp-net-core-applications-to-kubernetes-part-11-avoiding-downtime-in-rolling-deployments-by-blocking-sigterm/

In this post I describe an issue I experienced with the NGINX ingress controller where you get 502 errors during a rolling deployment, and how I fixed it

https://andrewlock.net/deploying-asp-net-core-applications-to-kubernetes-part-11-avoiding-downtime-in-rolling-deployments-by-blocking-sigterm/

**Comment by kjeske** [comment]
That was super helpful, thank you!

[a comment by an automated account is not shown]
